xml/ns-System.Runtime.Intrinsics.X86.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Runtime.Intrinsics.X86">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains classes representing the different SIMD instruction sets available to the x86 architecture. Not all of these instructions sets are available on all x86 CPUs though. Check the "IsSupported" property on each class to determine compatibility with the executing CPU.</summary>
```

Comment:
Not sure if it's worth clarifying that x86 includes x64 in this context.

Reply:
I added x64 in my latest commit.